README.md

```diff
@@ -22,11 +22,28 @@ And then execute:
 
 You can instantiate a number of SharePoint clients in your application:
 
+#### Token authentication
+
 ```rb
 client = Sharepoint::Client.new({
-  username: 'username',
-  password: 'password',
-  uri: 'https://sharepoint_url'
+    authentication: "token",
+    client_id: "client_id",
+    client_secret: "client_secret",
+    tenant_id: "tenant_id",
+    cert_name: "cert_name",
+    auth_scope: "auth_scope",
+    uri: "http://sharepoint_url"
```

Comment:
please keep using 2 space indentation also in `.md` files

Reply:
fixed